networkx/classes/digraph.py

```diff
@@ -855,7 +855,7 @@ def edges(self):
         Parameters
         ----------
         nbunch : single node, container, or all nodes (default= all nodes)
-            The view will only report edges incident to these nodes.
+            The view will only report edges from these nodes (outgoing if directed).
```

Comment:
The "(outgoing if directed)" seems unnecessary here, since this is in the docstring for the DiGraph class (so "if directed" will always be true).